plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -558,7 +558,24 @@ private void cloneFromSCM(PomData pomData, String name, String version, File che
         ScmRepository repository = scmManager.makeScmRepository(pomData.getConnectionUrl());
         CheckOutScmResult result = scmManager.checkOut(repository, new ScmFileSet(checkoutDirectory), new ScmTag(scmTag));
 
-        if (!result.isSuccess()) {
+        if (!result.isSuccess() && config.getAlternativePluginOrganization() != null) {
+            System.out.println("Using alternative organization in github");
+            if (checkoutDirectory.isDirectory()) {
+                FileUtils.deleteDirectory(checkoutDirectory);
+            }
+
+            Pattern pattern = Pattern.compile("(.*/github.com/)([^/]*)(.*)");
+            Matcher matcher = pattern.matcher(pomData.getConnectionUrl());
+            matcher.find();
+            String connectionURL = matcher.replaceFirst("$1" + config.getAlternativePluginOrganization() + "$3");
```

Comment:
It would be nice to log the URL used to understand where the source is being downloaded from.

Reply:
Thanks!! included